src/data/battler-tags/ability-battler-tag.ts

```diff
@@ -0,0 +1,23 @@
+import { BattlerTag } from "#app/data/battler-tags/battler-tag";
+import type { Abilities } from "#enums/abilities";
+import type { BattlerTagLapseType } from "#enums/battler-tag-lapse-type";
+import type { BattlerTagType } from "#enums/battler-tag-type";
+
+export class AbilityBattlerTag extends BattlerTag {
```

Comment:
This should probably be `abstract`?
```suggestion
export abstract class AbilityBattlerTag extends BattlerTag {
```